packages/frontend/src/pages/SqlRunnerNew.tsx

```diff
@@ -1,39 +1,75 @@
+import { IconDatabase } from '@tabler/icons-react';
 import { useState } from 'react';
 import { useParams } from 'react-router-dom';
 import MantineIcon from '../components/common/MantineIcon';
 import Page from '../components/common/Page/Page';
 import { Sidebar } from '../features/sqlRunner';
+import { ContentPanel } from '../features/sqlRunner/components/ContentPanel';
+import { Header } from '../features/sqlRunner/components/Header';
+import { RightSidebar } from '../features/sqlRunner/components/RightSidebar';
 
 const SqlRunnerNewPage = () => {
     const params = useParams<{ projectUuid: string }>();
     const selectedProjectUuid = params.projectUuid;
-    const [isSidebarOpen, setSidebarOpen] = useState(true);
+    const [isLeftSidebarOpen, setLeftSidebarOpen] = useState(true);
+    const [isRightSidebarOpen, setRightSidebarOpen] = useState(false);
 
     return (
         <Page
             title="SQL Runner"
-            withFullHeight
-            withPaddedContent
-            isSidebarOpen={isSidebarOpen}
+            noContentPadding
+            header={<Header />}
+            isSidebarOpen={isLeftSidebarOpen}
             sidebar={
                 <Sidebar
                     projectUuid={selectedProjectUuid}
-                    setSidebarOpen={setSidebarOpen}
+                    setSidebarOpen={setLeftSidebarOpen}
                 />
             }
+            isRightSidebarOpen={isRightSidebarOpen}
+            rightSidebar={<RightSidebar setSidebarOpen={setRightSidebarOpen} />}
         >
-            <Group>
-                {!isSidebarOpen && (
-                    <ActionIcon size="xs">
-                        <MantineIcon
-                            icon={IconArrowRight}
-                            onClick={() => setSidebarOpen(true)}
-                        />
-                    </ActionIcon>
+            <Group
+                align={'stretch'}
+                grow
+                spacing="none"
+                p={0}
+                style={{ flex: 1 }}
+            >
+                {!isLeftSidebarOpen && (
+                    <Paper
+                        shadow="none"
+                        radius={0}
+                        px="md"
+                        py="lg"
+                        withBorder
+                        style={{ flexGrow: 0 }}
+                    >
+                        <Tooltip
+                            variant="xs"
+                            label={'Open sidebar'}
+                            position="right"
+                        >
+                            <ActionIcon size="xs">
+                                <MantineIcon
+                                    icon={IconDatabase}
+                                    onClick={() => setLeftSidebarOpen(true)}
+                                />
+                            </ActionIcon>
+                        </Tooltip>
+                    </Paper>
                 )}
```

Comment:
todo: update <Page/> to config sidebar closed width and remove this from here

Reply:
I don't think we can do it with the Transition component. We should do this work after "Replace Sidebar's Transition with an equivalent animation component [#10571](https://github.com/lightdash/lightdash/issues/10571)"